README.md

```diff
@@ -25,6 +25,9 @@ Add this to your `pom.xml`:
 
 Then add the appender to your `logback.xml`.
 
+If not using Maven, download [logback-syslog4j-1.0.0.jar][] and
+[syslog4j-0.9.46-bin.jar][] (or latest).  Place these files in the classpath,
```

Comment:
Maybe swap `or latest` with `or newer`? I don't know. That's all I could think of and it's pretty weak, I'll admit.